mava/types.py

```diff
@@ -96,6 +96,20 @@ class PPOTransition(NamedTuple):
     info: Dict
 
 
+class RNNPPOTransition(NamedTuple):
```

Comment:
Nit: too many capitals, just a suggestion, but at least for RNN->Rnn :joy: 
```suggestion
class RnnPpoTransition(NamedTuple):
```

Reply:
I prefer the capitals - keen to hear thoughts of others.

Reply:
I agree with @RuanJohn